src/mam4xx/aero_model.hpp

```diff
@@ -810,6 +835,171 @@ bool examine_prec_exist(const int level_for_precipitation, const Real pdel[],
   return isprx;
 }
 
+// =============================================================================
+KOKKOS_INLINE_FUNCTION
+void set_f_act_coarse(const int kk,
+                      const Diagnostics::ColumnTracerView &state_q,
+                      const Diagnostics::ColumnTracerView &ptend_q,
+                      const Real dt, Real &f_act_conv_coarse,
+                      Real &f_act_conv_coarse_dust,
+                      Real &f_act_conv_coarse_nacl) {
+  // -----------------------------------------------------------------------
+  //  set the mass-weighted sol_factic for coarse mode species
+  // -----------------------------------------------------------------------
+  // clang-format off
+  /*
+  in  :: kk;
+       state_q and ptend_q only use dust and seasalt in this subroutine
+  in  :: state_q    ! tracer of state%q [kg/kg]
+  in  :: ptend_q    ! tracer tendency (ptend%q) [kg/kg/s]
+  in  :: dt         ! time step [s]
+  out :: f_act_conv_coarse      ! prescribed coarse mode aerosol activation fraction for convective
+  out :: f_act_conv_coarse_dust ! prescribed dust aerosol activation fraction for convective cloud [fracti
+  out :: f_act_conv_coarse_nacl ! prescribed seasalt aerosol activation fraction for convective cloud [fra
```

Comment:
Did the two lines above get truncated in a copy-paste?

Reply:
Yes, probably a cut/paste error without text wrapping turned on.